seacrowd/sea_datasets/visim_400/visim_400.py

```diff
@@ -0,0 +1,181 @@
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_t2t",
+            version=_SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema="seacrowd_t2t",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+
+            features = datasets.Features(
+                {
+                    "Word1": datasets.Value("string"),
+                    "Word2": datasets.Value("string"),
+                    "POS": datasets.Value("string"),
+                    "Sim1": datasets.Value("string"),
+                    "Sim2": datasets.Value("string"),
+                    "STD": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        data_dir = dl_manager.download_and_extract(_URLS[_DATASETNAME])
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, f"ViData/ViSim-400/Visim-400.txt"),
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        with open(filepath, "r", encoding="utf-8") as file:
+            lines = file.readlines()
+
+        data = []
+        for line in lines:
+            columns = line.strip().split("\t")
+            data.append(columns)
+
+        df = pd.DataFrame(data[1:], columns=data[0])
+
+        for index, row in df.iterrows():
+
+            if self.config.schema == "source":
+                example = row.to_dict()
+
+            elif self.config.schema == "seacrowd_t2t":
+
+                example = {
+                    "id": str(index),
+                    "text_1": str(row["Word1"]),
+                    "text_2": str(row["Word2"]),
+                    "text_1_name": str(row["Sim1"]),
+                    "text_2_name": str(row["Sim2"]),
+                }
+
+            yield index, example
+
+
+# This template is based on the following template from the datasets package:
+# https://github.com/huggingface/datasets/blob/master/templates/new_dataset_script.py
```

Comment:
Remove this.

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/9e66e8a8f72b1d791ebc0d73b014f3fa92db0828.